libraries/transfer/src/main/java/org/dataportabilityproject/transfer/ImageStreamProvider.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.transfer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ImageStreamProvider {
+
+  /**
+   * Gets an input stream to an image, given its URL.
+   */
+  public InputStream get(String urlStr) throws IOException {
```

Comment:
Just curious, why can't this be a static utility function?

Reply:
So I can mock it.